app/controllers/amr_heatmap_controller.rb

```diff
@@ -0,0 +1,45 @@
+class AmrHeatmapController < ApplicationController
+  include ReportHelper
+
+  before_action :admin_required
+
+  current_power do # Put this here for CLI
+    Power.new(current_user)
+  end
+
+  power :viewable_samples, as: :samples_scope
```

Comment:
You should be able to use `current_power.viewable_samples` directly without re-assigning to `samples_scope`.

Reply:
Done!